### ISSUE TITLE
Remix logs unusable from 500-item JSON object logged by performance.checker.ts

### ISSUE THREAD

**Issue by jdempcy** [AEPKILL/devtools-detector#51]
In Remix apps the console log of the repeated 500-item JSON object logged by performance.checker.ts is put into the server logs, rendering them unusable (since console.clear() does nothing there) filling up over 10,000 lines in every page load.

```
    '480': '480',
    '481': '481',
    '482': '482',
    '483': '483',
    '484': '484',
    '485': '485',
    '486': '486',
    '487': '487',
    '488': '488',
    '489': '489',
    '490': '490',
    '491': '491',
    '492': '492',
    '493': '493',
    '494': '494',
    '495': '495',
    '496': '496',
    '497': '497',
    '498': '498',
    '499': '499'
```

etc, looping over and over.

I have not been able to fix this by overriding devtoolsDetector.launch() as advised in another issue (of console clearing). That solution did not work. Is there a way to add an option to devtools-detector to disable this performance.checker.ts logging?

**Comment by jdempcy** [comment]
This is Remix: https://github.com/remix-run/remix/discussions/4599

For a reproducible demo, run:

```npx create-remix@latest --template remix-run/indie-stack blog-tutorial```
Then, to run the app:

```npm run dev```

Confirm that the logs look good. Then stop the server, run `npm i devtools-detector`, and then import it anywhere in the project. Bam, your logs will be full of 20,000 lines of JSON object from performance.checker.ts on every request.

**Comment by AEPKILL** [comment]
Maybe fixed, Please try install devtools-detector@2.0.14

**Comment by jdempcy** [comment]
Looks like that fixed it! Thank you so much AEPKILL!
